setup.cfg

```diff
@@ -6,12 +6,12 @@ author_email = adam-r-j@hotmail.com
 description = Functions for simulating self-shading of two-axis trackers
```

Comment:
Should we be consistent about the package description?

Currently, there are several different descriptions, e.g., here, in the README, and on ReadTheDocs.

For example:
> twoaxistracking is a python package for simulating two-axis tracking solar collectors, particularly self-shading.

Reply:
+1 to being consistent and your proposed description.  I'll update this PR with that.

For reference, here are the existing descriptions I've found:

setup.cfg:
> Functions for simulating self-shading of two-axis trackers

README.md:
> "twoaxistracking" is a python package for simulating self-shading in fields of two-axis trackers.

GitHub "About":
>  Functions for simulation shading in fields of two-axis solar trackers. 

readthedocs:
>  twoaxistracking is a python package for simulating the performance of two-axis tracking solar collectors, particularly for calculating self-shading.